docs/src/test/scala/docs/http/scaladsl/server/FullSpecs2TestKitExampleSpec.scala

```diff
@@ -53,8 +53,8 @@ class FullSpecs2TestKitExampleSpec extends Specification with Specs2RouteTest {
     "return a MethodNotAllowed error for PUT requests to the root path" in {
       // tests:
       Put() ~> Route.seal(smallRoute) ~> check {
-        status === StatusCodes.MethodNotAllowed
-        responseAs[String] === "HTTP method not allowed, supported methods: GET"
+        status shouldEqual StatusCodes.MethodNotAllowed
```

Comment:
I used shouldEqual because it is in other samples, for example in FullTestKitExampleSpec, so it should be replaced there and in other places too. What do you think is better?